util-core/src/main/scala/com/twitter/util/StorageUnit.scala

```diff
@@ -126,7 +126,7 @@ class StorageUnit(val bytes: Long) extends Ordered[StorageUnit] {
   def max(other: StorageUnit): StorageUnit =
     if (this > other) this else other
 
-  override def toString(): String = inBytes + ".bytes"
+  override def toString(): String = s"$inBytes bytes"
```

Comment:
for the format, or for the interpolator? I missed the `.` there, that change was unintentional, but the interpolator is there because any2stringadd is on its way out, and should be either `s"$inBytes.bytes"` with, or `inBytes.toString + ".bytes"` without interpolator

Reply:
gotcha, I will only add the `.` then.